extensions/wikia/AchievementsII/Ach_setup.php

```diff
@@ -211,10 +211,10 @@ function AchAjax() {
 
 function Ach_UserPreferences( $user, &$preferences ) {
 	global $wgEnableUserPreferencesV2Ext;
-	$section = (!empty($wgEnableUserPreferencesV2Ext)) ? 'under-the-hood/advanced-displayv2' : 'misc';
+	$section = ( !empty( $wgEnableUserPreferencesV2Ext ) ) ? 'under-the-hood/advanced-displayv2' : 'misc';
 
 	if( $user->isLoggedIn() ) {
-		$preferences['hidepersonalachievements'] = array(
+		$preferences[ 'hidepersonalachievements' ] = array(
 			'type' => 'toggle',
 			'label-message' => 'achievements-toggle-hide', // a system message
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)